scripts/e2e-test.sh

```diff
@@ -9,7 +9,7 @@ command -v gatsby-dev || command -v sudo && sudo npm install -g gatsby-dev-cli |
 # setting up child integration test link to gatsby packages
 cd "$SRC_PATH" &&
 gatsby-dev --set-path-to-repo "$GATSBY_PATH" &&
-gatsby-dev --scan-once --quiet && # copies _all_ files in gatsby/packages
+gatsby-dev --scan-once  && # copies _all_ files in gatsby/packages
```

Comment:
```suggestion
gatsby-dev --scan-once  --quiet && # copies _all_ files in gatsby/packages
```

Reply:
I think this change should stay — it's hiding errors that happen